src/connect-params.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import {
+  coerce,
+  createMapperFunc,
+  get,
+  mapValues,
+  noop,
+  wrapDisplayName,
+} from './utils'
+import { withRouter } from 'react-router-v4'
```

Comment:
I'm concerned that this won't work with how I've setup `peerDependencies` and `devDependencies`